staker-rewards/rewards.js

```diff
@@ -49,7 +57,36 @@ async function run() {
 
     const operatorsRewards = await calculateOperatorsRewards(context, interval)
 
-    console.table(operatorsRewards)
+    if (process.env.OUTPUT_MODE === "text") {
+      const format = {
+        groupSeparator: "",
+        decimalSeparator: ".",
+      }
+
+      operatorsRewards.forEach((operatorRewards) =>
+        console.log(
+          `${operatorRewards.operator}
+           ${operatorRewards.isFraudulent} 
+           ${operatorRewards.keygenCount}
+           ${operatorRewards.keygenFailCount} 
+           ${operatorRewards.keygenSLA} 
+           ${operatorRewards.signatureCount} 
+           ${operatorRewards.signatureFailCount}
+           ${operatorRewards.signatureSLA} 
+           ${operatorRewards.keepStaked.toFormat(format)} 
+           ${operatorRewards.ethBonded.toFormat(format)} 
+           ${operatorRewards.ethUnbonded.toFormat(format)}
+           ${operatorRewards.ethTotal.toFormat(format)} 
+           ${operatorRewards.ethScore.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.boost.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.rewardWeight.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.totalRewards.toFormat(decimalPlaces, format)}
```

Comment:
1 KEEP is displayed as `1000000000000000000` here, correct? If so, let's drop decimal separator and decimal places altogether so that e.g. `1000000000000000000.99` becomes `1000000000000000000`.